debian/changelog

```diff
@@ -1,3 +1,10 @@
+matrix-synapse-py3 (1.29.0) UNRELEASED; urgency=medium
+
+  [ Jonathan de Jong ]
+  * Remove the python -B flag (don't generate bytecode) in scripts and documentation.
+
+ -- Synapse Packaging team <packages@matrix.org>  Fri, 26 Feb 2021 14:41:31 +0100
```

Comment:
i don't exactly know how to "correctly" annotate or sign this, please suggest how i might change this to something that works better

Reply:
This looks correct to me :slightly_smiling_face: We'll update it with the `New synapse release 1.29.0.` bit during release.